migrator/migrations/m_172_to_m_173_enable_search_on_api_tokens/migration.go

```diff
@@ -0,0 +1,64 @@
+package m172tom173
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	frozenSchema "github.com/stackrox/rox/migrator/migrations/frozenschema/v75"
+	newAPITokenStore "github.com/stackrox/rox/migrator/migrations/m_172_to_m_173_enable_search_on_api_tokens/newapitokenpostgresstore"
+	oldAPITokenStore "github.com/stackrox/rox/migrator/migrations/m_172_to_m_173_enable_search_on_api_tokens/oldapitokenpostgresstore"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"gorm.io/gorm"
+)
+
+const (
+	batchSize = 500
+
+	startSeqNum = 172
+)
+
+var (
+	migration = types.Migration{
+		StartingSeqNum: startSeqNum,
+		VersionAfter:   &storage.Version{SeqNum: int32(startSeqNum + 1)},
+		Run: func(database *types.Databases) error {
+			return migrateAPITokens(database.PostgresDB, database.GormDB)
+		},
+	}
+)
+
+func init() {
+	migrations.MustRegisterMigration(migration)
+}
+
+// Additional code to support the migration
+
+func migrateAPITokens(postgresDB *postgres.DB, gormDB *gorm.DB) error {
+	ctx := context.Background()
+	oldStore := oldAPITokenStore.New(postgresDB)
+	newStore := newAPITokenStore.New(postgresDB)
+	pgutils.CreateTableFromModel(ctx, gormDB, frozenSchema.CreateTableAPITokensStmt)
```

Comment:
nit: logistically I think we should create the table before we create create the new store.  Doesn't really matter since you aren't using the store until after the table is created though.